src/libraries/Native/Unix/System.Globalization.Native/CMakeLists.txt

```diff
@@ -57,24 +57,32 @@ set(NATIVEGLOBALIZATION_SOURCES
     pal_icushim.c
 )
 
+if (NOT EXCLUDE_VERSION_HEADER_FILE)
+    list(APPEND NATIVEGLOBALIZATION_SOURCES
```

Comment:
This should be excluded for all static builds. I do not think we need `EXCLUDE_VERSION_HEADER_FILE`.

Reply:
You mean, `NO GEN_SHARED_LIB`?

Reply:
I mean that it should be in the `add_library(System.Globalization.Native ...` target as it is today, and deleted from the `add_library(System.Globalization.Native-Static` target.

Reply:
So `System.Globalization.Native-Static` shouldn’t carry the version information?

Reply:
I do not see a good reason for it.

Reply:
Ok then I’ll change it to that.